test/wasm2c/add.txt

```diff
@@ -98,6 +76,17 @@ u32 w2c_test_add(w2c_test*, u32, u32);
 
 #define UNREACHABLE TRAP(UNREACHABLE)
 
+typedef uint8_t u8;
```

Comment:
Having these definitions in the source is a bit confusing now. On the one hand, we have functions using the C types for params and returns. While these short types seem to be defined exclusively for params in DEFINE_LOAD, DEFINE_STORE etc.  Does it make sense to eliminate the short types altogether by also changing the params of DEFINE_LOAD and DEFINE_STORE.

Reply:
We certainly can if that's what people want! It would be a more intrusive change, and it seemed mostly harmless (to me) to keep these typedefs in the generated .c file (vs. leaking them into the embedder's namespace). But I don't have a strong view one way or another if people prefer to just bite the bullet now.

Reply:
I think we can probably just remove it tbh. But I guess if the main goal was to avoid leaking the types into to the embedder, I'm fine either way